tools/train.py

```diff
@@ -154,6 +156,13 @@ def main():
     cfg_samples_per_gpu = cfg.data.samples_per_gpu
     if args.update_config is not None:
         cfg.merge_from_dict(args.update_config)
+    if args.tensorboard_dir is not None:
+        hooks = [hook for hook in cfg.log_config.hooks if hook.type == 'TensorboardLoggerHook']
+        if hooks:
+            hooks[0].log_dir = args.tensorboard_dir
+        else:
+            logging.warning('Failed to find TensorboardLoggerHook')
```

Comment:
Is it possible to use the logger that is defined and initialized at line 203?

Reply:
done